packages/main/src/TableRowBase.ts

```diff
@@ -1,10 +1,10 @@
 import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { isEnter, isSpace } from "@ui5/webcomponents-base/dist/Keys.js";
```

Comment:
do we still them?

Reply:
Yes. As the Space, Enter logic is still handled in the TableRow (as mentioned in a comment above), imports are still valid.